src/Agent.Plugins/GitCliManager.cs

```diff
@@ -565,7 +565,7 @@ private async Task<int> ExecuteGitCommandAsync(AgentTaskPluginExecutionContext c
             string arg = StringUtil.Format($"{command} {options}").Trim();
             context.Command($"git {arg}");
 
-            var processInvoker = new ProcessInvoker(context);
+            var processInvoker = new ProcessInvoker(context, true);
```

Comment:
Nit: can we make this a named parameter here to make it easier to see what's going on:

`var processInvoker = new ProcessInvoker(context, disallowWorkerCommands: true);`

Same for occurrences of it below.